src/software/SfMWebGLViewer/common/main.js

```diff
@@ -245,7 +245,7 @@ function onMouseMove( e )
       var new_rad = d * Math.sin( arad * 0.5 ); 
       trackball.setRadius( new_rad * 0.8 ); 
     }
-    else if (e.altKey )
+    else if (e.altKey || e.ctrlKey)
```

Comment:
Which is your motivation behind adding this key?
Does another software you are using can use ctrl to pan?

Reply:
See commit message:

> On Linux systems like Ubuntu with Gnome or Unity, alt+drag is bound to
move the window, so this alternative allows panning also in that
situation.